packages/reporting-server/README.md

```diff
@@ -98,6 +156,33 @@ for more information, see https://tortoise-orm.readthedocs.io/en/latest/database
 npm run test
 ```
 
+### Running migration tests locally
```

Comment:
Would it be possible to add a npm script to run migration tests?

Reply:
Sure

Reply:
I created a script to build the docker container, run it, and apply migrations. 
QQ: I need to wait for Postgres to be up running before running the migrations. 
I tried (all trying to connect from my terminal/bash to docker container):
* There is something called Health in docker, but with this command, I can only check is the container is running but not if Postgres is running. 
* I tried to curl `dockerIp:port`
* pg_isready -d <db_name> -h <host_name> -p <port_number> -U <db_user>                      

What do you think is the best way?

There is an alternative that I did not try yet, and is to make the script inside the container to check is the database is up and running before migration. 

PS: Now it is just with a sleep (it's working but is not ideal.)

Reply:
I believe the example deployment has some way to determine if postgres is ready, could we use that here?

Reply:
Yes, but that Postgres DB is running in a k8s pod. I'll keep digging.